example.php

```diff
@@ -1,7 +1,14 @@
-<?php
-
-$foo = NULL;
-// is_null is a function that does stuff 
-is_null($foo);
-
-?>
\ No newline at end of file
+// is_null your comment
<// is_null your comment
?// is_null your comment
p// is_null your comment
h// is_null your comment
p// is_null your comment

```

Comment:
```diff
- // is_null your comment
+ // is_null {{ .funcComment }}
```

Change `{{ .funcComment }}` by commenting: `@CodeLingoBot rewrite set funcComment "your comment"`

Reply:
@CodelingoBotTester rewrite set funcComment "this is a test"